buildbot/configure.py

```diff
@@ -198,6 +201,13 @@ def do_configure(args):
             "-DSYCL_LIBCXX_INCLUDE_PATH={}".format(args.libcxx_include),
             "-DSYCL_LIBCXX_LIBRARY_PATH={}".format(args.libcxx_library)])
 
+    if args.default_openmp:
+        omp_dir = os.path.join(abs_src_dir, "openmp")
+        cmake_cmd.extend([
+            "-DLLVM_EXTERNAL_OPENMP_SOURCE_DIR={}".format(omp_dir),
```

Comment:
Why isn't line 96 enough?

Reply:
Hi, it is a good question. Let me check without this line.

Reply:
Hi @aelovikov-intel and @bader, it turns out that `"-DLLVM_EXTERNAL_OPENMP_SOURCE_DIR={}".format(omp_dir)` is required. 

If I compile dpc++ without and try to configure any cmake project which contains `find_package(OpenMP REQUIRED)`  then I will get the following cmake-error

```
CMake Error at /nfs/c3po/software/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/cmake-3.23.0-jhleqvqh73yiba32hes3rjahq42dl7db/share/cmake-3.23/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find OpenMP_C (missing: OpenMP_C_FLAGS OpenMP_C_LIB_NAMES)
Call Stack (most recent call first):
  /nfs/c3po/software/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/cmake-3.23.0-jhleqvqh73yiba32hes3rjahq42dl7db/share/cmake-3.23/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
  /nfs/c3po/software/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/cmake-3.23.0-jhleqvqh73yiba32hes3rjahq42dl7db/share/cmake-3.23/Modules/FindOpenMP.cmake:545 (find_package_handle_standard_args)
  /nfs/c3po/software/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/cmake-3.23.0-jhleqvqh73yiba32hes3rjahq42dl7db/share/cmake-3.23/Modules/CMakeFindDependencyMacro.cmake:47 (find_package)
  /nfs/c3po/software/spack/opt/spack/linux-ubuntu20.04-x86_64/gcc-9.3.0/easi-1.1.2-aoypioxcgozdkco422xajef62tix234s/lib/cmake/easi/easiConfig.cmake:32 (find_dependency)
  CMakeLists.txt:187 (find_package)
```

It happens because OpenMP has not been installed. Here is the result when I am trying to compile a toy-openmp program:
```
$ cat ./main.cpp
#include <iostream>
#include <vector>
#include <omp.h>

int main() {
  std::vector<float> vec(0, 1024);

  #pragma omp parallel for
  for (size_t i = 0; i < vec.size(); ++i) {
    vec[i] = static_cast<float>(i);
  }

  return 0;
}

$ clang++ -fopenmp=libiomp5 ./main.cpp
./main.cpp:3:10: fatal error: 'omp.h' file not found
#include <omp.h>
```

All in all, `"-DLLVM_EXTERNAL_OPENMP_SOURCE_DIR={}".format(omp_dir)` is required